etsin_finder/frontend/js/components/qvain/files/selectedFiles.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { Component, Fragment } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { inject, observer } from 'mobx-react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCopy, faFolder } from '@fortawesome/free-solid-svg-icons'
+import Translate from 'react-translate-component'
+import { ButtonGroup, ButtonLabel, EditButton, DeleteButton } from '../general/buttons'
+import FileForm from './fileForm'
+import DirectoryForm from './directoryForm'
+import { getDirectories, getFiles } from '../utils/fileHierarchy'
+
+export class SelectedFilesBase extends Component {
```

Comment:
Used to visualize all the directories and files that the user has selected with the ```FileSelector```. Houses the Form components for directories and files and renders the buttons for interacting with the selections.